tests/test_server.py

```diff
@@ -635,9 +636,9 @@ def test_manual(runestone_db_tools, test_user):
     ('default/terms', False, 'Terms and Conditions', 1),
     ('default/privacy', False, 'Runestone Academy Privacy Policy', 1),
     ('default/donate', False, 'Support Runestone Interactive', 1),
-    # FIXME: This produces an exception.
-    #('default/coursechooser', True, 'xxx', 1),
-    #('default/removecourse', True, 'xxx', 1),
+    # TODO: This soesn't really test the body of either of these
```

Comment:
I agree. This is just a rough test, and should be replaced with detailed tests. A lot of these tests are very surface and need expanding.